lua/telescope/command.lua

```diff
@@ -0,0 +1,90 @@
+local themes = require('telescope.themes')
+local builtin = require('telescope.builtin')
+local extensions = require('telescope._extensions').manager
+local config = require('telescope.config')
+local command = {}
+
+-- convert command line string arguments to
+-- lua number boolean type and nil value
+local function convert_user_opts(user_opts)
+  local default_opts = config.values
+
+  local _switch = {
+    ['boolean'] = function(key,val)
+      if val == 'false' then
+        user_opts[key] = false
+      end
+      user_opts[key] = true
+    end,
+    ['number'] = function(key,val)
+      user_opts[key] = tonumber(val)
+    end,
+    ['string'] = function(key,val)
+      if val == 'nil' then
+        user_opts[key] = nil
+      end
+    end
+  }
+
+  local _switch_metatable = {
+    __index = function(_,k)
+      print(string.format('Type of %s does not match',k))
+    end
+  }
```

Comment:
Should we fail silent? Its not really useful for end user and more just helpful for debugging now

Reply:
yep. just for debug we never know that what arguments that user passed.